File: terraform/lambdas/hello/handler.py
import sys
import os
from datetime import datetime

sys.path.insert(0, os.path.join(os.path.dirname(__file__), '..'))

from shared.response import success_response

def handler(event, context):
    return success_response(200, {
        'message': 'Hello from Styrkr API',
        'requestId': context.aws_request_id,
        'timestamp': datetime.utcnow().isoformat() + 'Z'
    })

File: terraform/lambdas/hello/handler.ts
import type { 
  APIGatewayProxyEventV2WithJWTAuthorizer, 
  APIGatewayProxyResultV2, 
  Context 
} from "aws-lambda";
import { successResponse } from "../shared/response.js";

export const handler = async (
  event: APIGatewayProxyEventV2WithJWTAuthorizer,
  context: Context
): Promise<APIGatewayProxyResultV2> => {
  return successResponse(200, {
    message: "Hello from Styrkr API",
    requestId: context.awsRequestId,
    timestamp: new Date().toISOString(),
  });
};



